beams/beams/custom_scripts/attendance/attendance.py

```diff
@@ -22,17 +24,40 @@ def send_absence_reminder():
                     "docstatus": 1,
                 }
             )
+
+            # If no leave exists, create a leave application
             if not leave_exists:
+                # Fetch the leave type from the leave allocation
+                leave_type = frappe.db.get_value(
+                    "Leave Allocation",
+                    {
+                        "employee": employee["employee"],
+                        "from_date": ("<=", employee["attendance_date"]),
+                        "to_date": (">=", employee["attendance_date"]),
+                        "docstatus": 1
+                    },
+                    "leave_type"
```

Comment:
leave type should be the penalty leave type as configured in Employment Type according to the task, fetching it like this will fetch a random leave type